docs/tutorial/part_2.rst

```diff
@@ -254,9 +254,28 @@ tutorial.) Then the connection will be given to the ``URLRouter``.
 The ``URLRouter`` will examine the HTTP path of the connection to route it to a
 particular consumer, based on the provided ``url`` patterns.
 
-Let's verify that the consumer for the ``/ws/chat/ROOM_NAME/`` path works. Start the
-Channels development server::
-
+Let's verify that the consumer for the ``/ws/chat/ROOM_NAME/`` path works. Run migrations to
+apply database changes(we need database to manage session) and then start the Channels development
```

Comment:
Can you put a space before the `(` and change the contents to `(Django's session framework needs the database)`?

Reply:
Sure.